db/rmWrapper.py

```diff
@@ -445,7 +447,7 @@ def get_near_gyms(self, lat, lng, hash, raid_no, dist, unique_hash="123"):
             ") "
             "AS distance, latitude, longitude, name, description, url "
             "FROM gym "
-            "LEFT JOIN gymdetails ON gym.gym_id = gymdetails.gym_id "
+            "LEFT JOIN gym_details ON gym.gym_id = gym_details.gym_id "
```

Comment:
Shouldn't this be gymdetails and not gym_details for RM's database?

```suggestion
            "LEFT JOIN gymdetails ON gym.gym_id = gymdetails.gym_id "
```

Reply:
That isn't part of my commit. Not sure why that's in there. Snoopy fixed it last commit